nav2_util/include/nav2_util/simple_action_server.hpp

```diff
@@ -150,6 +150,9 @@ class SimpleActionServer
     std::lock_guard<std::recursive_mutex> lock(update_mutex_);
 
     if (!server_active_) {
+      RCLCPP_INFO(
+        node_logging_interface_->get_logger(),
+        "Action server is inactive. Rejecting the goal.");
```

Comment:
So actually this statement is not true then?

```
@brief handle the goal requested: accept or reject. This implementation always accepts.
```

Reply:
https://github.com/ros-navigation/navigation2/pull/5150